core/Tracker/TrackerCodeGenerator.php

```diff
@@ -245,16 +245,29 @@ private function getJavascriptTagOptions($idSite, $mergeSubdomains, $mergeAliasU
         $websiteHosts = array();
         $firstHost = null;
         foreach ($websiteUrls as $site_url) {
+            if (empty($site_url)) {
+                continue;
+            }
+            
             $referrerParsed = parse_url($site_url);
 
             if (!isset($firstHost) && isset($referrerParsed['host'])) {
                 $firstHost = $referrerParsed['host'];
             }
 
-            $url = $referrerParsed['host'];
+            if (isset($referrerParsed['host'])) {
+                $url = $referrerParsed['host'];
+            } else {
+                $url = '';
+            }
             if (!empty($referrerParsed['path'])) {
                 $url .= $referrerParsed['path'];
             }
+            
+            if (!empty($url)) {
+                $websiteHosts[] = $url;
+            }
+            
             $websiteHosts[] = $url;
```

Comment:
shouldn't that line be removed? otherwise each url would be added twice.

Reply:
👍 fixed